prowler/providers/aws/aws_provider.py

```diff
@@ -491,30 +491,23 @@ def print_credentials(self):
         profile = (
             self._identity.profile if self._identity.profile is not None else "default"
         )
-        # TODO: rename AWS Filter Region to AWS Regions, and UserId to User ID
-        # review new banner
-        #       report = f"""
-        # The current audit for AWS will use the following credentials:
-
-        # CLI Profile: {Fore.YELLOW}[{profile}]{Style.RESET_ALL} Regions: {Fore.YELLOW}[{regions}]{Style.RESET_ALL}
-        # Account: {Fore.YELLOW}[{self._identity.account}]{Style.RESET_ALL} User ID: {Fore.YELLOW}[{self._identity.user_id}]{Style.RESET_ALL}
-        # Caller Identity ARN: {Fore.YELLOW}[{self._identity.identity_arn}]{Style.RESET_ALL}
-        # """
-        report = f"""
-This report is being generated using credentials below:
-
-AWS-CLI Profile: {Fore.YELLOW}[{profile}]{Style.RESET_ALL} AWS Filter Region: {Fore.YELLOW}[{regions}]{Style.RESET_ALL}
-AWS Account: {Fore.YELLOW}[{self._identity.account}]{Style.RESET_ALL} UserId: {Fore.YELLOW}[{self._identity.user_id}]{Style.RESET_ALL}
-Caller Identity ARN: {Fore.YELLOW}[{self._identity.identity_arn}]{Style.RESET_ALL}
-"""
+        report_lines = [
```

Comment:
Maybe not for now because is a nice to have, but this should be a "mandatory" attribute like the others the providers have to enforce including this when creating a new one